### PR TITLE
Fix code block style (MD040)

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) fixes a few errors related to code block style in the config guide and a few other random topics. 

The MD040 rule requires that a language be specified for fenced code blocks. However, it also identifies indented code blocks, so you'll see changes in this PR that replaces indentation with fences.

This is the first of many PRs related to #5252.

### CODE REVIEW

File: guides/v2.2/config-guide/cli/config-cli-subcommands-spt-util.md [hunk 1]
```diff
@@ -90,10 +90,10 @@ Run the following commands in the order shown to display the paths to the applic
 
 1.  Change to your Magento installation directory.
 
-  For example, `cd /var/www/magento2`
+    For example, `cd /var/www/magento2`
 
-	{:.bs-callout .bs-callout-info}
-  The commands run properly _only_ from your Magento installation directory.
+    {:.bs-callout .bs-callout-info}
```
Comment (hguthrie) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-spt-util.md]: In other files, you deleted the extra bs.callout.

File: README.md [hunk 2]
```diff
@@ -170,13 +170,13 @@ To build v2.0 documentation:
 
 1. Initialize the `2.0` branch in your local devdocs repository.
 
-    ```shell
+    ```bash
```
Comment (jcalcaben) [inline README.md]: What is the difference between: `bash`, `shell`, and `terminal`?

Comment (jeff-matthews) [reply README.md]: Shell and Terminal do not include the bash prompt `$`.

Screen Shot 2019-09-20 at 9 28 17 AM

Comment (jcalcaben) [reply README.md]: 👍

Comment (jeff-matthews) [reply README.md]: Sorry, to be clear, that's the difference in the files we render to HTML. In the README, i don't think it makes a difference.

Comment (dshevtsov) [reply README.md]:
````
```bash
echo 'Hello, world!'
```

```shell
echo 'Hello, world!'
```

```terminal
echo
```
````

Screen Shot 2019-09-20 at 09 32 52

File: guides/v2.2/advanced-reporting/report-xml.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ Columns are added using the `<attribute>` node.
 
 All report files are located in the `etc` directory of a module:
 
-```
+```terminal
```
Comment (dshevtsov) [inline guides/v2.2/advanced-reporting/report-xml.md]: This is not supposed to be a terminal output. For such cases, it makes sense to use the `text` [lexer](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers).

File: README.md (hunk 2 above)
Comment (shrielenee) [reply README.md]: Maybe it's common knowledge when to use each of these, but I'll need to brush up on when to use which---`bash`, `terminal`, `shell`.

File: guides/v2.2/advanced-reporting/report-xml.md (hunk 3 above)
Comment (jeff-matthews) [reply guides/v2.2/advanced-reporting/report-xml.md]: Fixed

File: README.md (hunk 2 above)
Comment (dshevtsov) [reply README.md]: Use `bash` for terminal commands when you want the `$` symbol to appear at the beginning.
Use `shell` for terminal commands when you don't need the `$` symbol.
Use `terminal` when you want to demonstrate terminal output.

````
To teach your machine some courtesy, run in your terminal:

```bash
echo 'Good day to you, Shrie!'
```

Enjoy a polite respond:

```terminal
Good day to you, Shrie!
```
````

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**